csv-exporter/index.js

```diff
@@ -27,9 +28,11 @@ const getAllDataSets = async () => {
     }
 
     const csv = new ObjectsToCsv(results);
+    const jsonFile = `test.json`;
     const fileToSave = `test.csv`;
 
     // Save to file:
+    await writeFile(`./${jsonFile}`, JSON.stringify(results));
```

Comment:
Make the JSON slightly nicer to read in case we ever need to read it.

```suggestion
    await writeFile(`./${jsonFile}`, JSON.stringify(results, null, '  '));
```